run/events-storage/anthos.md

```diff
@@ -0,0 +1,96 @@
+# Events for Cloud Run on Anthos – GCS tutorial
+
+This sample shows how to create a service that processes GCS events.
+
+## Setup
+
+Login to gcloud:
+
+```sh
+gcloud auth login
+```
+
+Configure project id:
+
+```sh
+gcloud config set project [PROJECT-ID]
+```
+
+Configure environment variables:
+
+```sh
+MY_RUN_SERVICE=gcs-service
+MY_RUN_CONTAINER=gcs-container
+MY_GCS_TRIGGER=gcs-trigger
+MY_GCS_BUCKET=gcs-bucket-$(gcloud config get-value project)
+```
+
+## Quickstart
+
+Set cluster name, location and platform:
+
+```sh
+gcloud config set run/cluster events-cluster
+gcloud config set run/cluster_location europe-west1-b
+gcloud config set run/platform gke
+```
+
+Deploy your Cloud Run service:
+
+```sh
+gcloud builds submit \
+ --tag gcr.io/$(gcloud config get-value project)/$MY_RUN_CONTAINER
+gcloud run deploy $MY_RUN_SERVICE \
```

Comment:
Initially got `ERROR: (gcloud.run.deploy) [Errno 61] Connection refused`

Successfully connected with `gcloud container clusters get-credentials events-cluster --zone us-central1-c --project {PROJECT_ID}` and ran the deploy command without issue

Reply:
I don't think the two are related. `get-credentials` sets up kubectl to talk to the cluster but it should not affect `gcloud run deploy`. The only thing that can affect deploy is the `run/platform gke` setting. Make sure it's set to `gke` to deploy to GKE/Anthos.